### PR TITLE
Updating the term "store" to "storage" in the CLI commands.

### PR DESCRIPTION
**Pull request by gilMars**
Changed the commands, `ml-git repository store add` and `ml-git repository store del` to deprecated.
New usage: `ml-git repository storage add` and `ml-git repository storage del`.

Output examples:
```
ml-git repository store add teste
[WARNING]: Deprecated command,  use 'storage' instead.
INFO - Admin: Add store [s3h://teste]

ml-git repository store del teste
[WARNING]: Deprecated command, use 'storage' instead.
INFO - Admin: Removed store [s3h://teste] from configuration file.
```

Help output:
```
ml-git repository --help
Usage: ml-git repository [OPTIONS] COMMAND [ARGS]...

  Management of this ml-git repository.

Options:
  --help  Show this message and exit.

Commands:
  config   Configuration of this ml-git repository
  gc       Cleanup unnecessary files and optimize the use of the disk space.
  init     Initialiation of this ml-git repository
  remote   Configure remote ml-git metadata repositories.
  storage  Storage management for this ml-git repository.
  store    [DEPRECATED]: Store management for this ml-git repository.
  update   This command will update all ml-entities metadata repository.
```

### CODE REVIEW

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -896,6 +901,60 @@ Example:
...
+                                  global level
+  --verbose                       Debug mode
+```
+
+Example:
+```
+$ ml-git repository storage add minio --endpoint-url=<minio-endpoint-url>
+```
+
+Use this command to add or delete a data storage to a ml-git project.
```
Comment (lucasncabral) [inline docs/mlgit_commands.md:934]:
```suggestion
Use this command to add a data storage to a ml-git project.
```

File: docs/mlgit_commands.md [hunk 2]
```diff
@@ -872,12 +875,14 @@ Example:
 $ ml-git repository store add minio --endpoint-url=<minio-endpoint-url>
 ```
 
-Use this command to add or delete a data store to a ml-git project.
+Use this command to add or delete a data storage to a ml-git project.
```
Comment (lucasncabral) [inline docs/mlgit_commands.md:878]:
```suggestion
Use this command to add a data storage to a ml-git project.
```

File: ml_git/commands/descriptor.py [hunk 3]
```diff
@@ -501,7 +507,56 @@
 
         'help': 'This command shows ml-entity-name\'s commit information like author, date, commit message.'
 
-    }
+    },
+
+    {
+        'name': 'add',
+
+        'callback': store.store_add,
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/descriptor.py]: Adjusted in 27a2458.

File: ml_git/commands/store.py [hunk 4]
```diff
@@ -3,43 +3,24 @@
...
 import ml_git.admin as admin
 from ml_git.commands.repository import repository
-from ml_git.commands.utils import set_verbose_mode
 
 
-@repository.group('store', help='Store management for this ml-git repository.', cls=DYMGroup)
+@repository.group('store', help='[DEPRECATED]: Storage management for this ml-git repository.', cls=DYMGroup)
 def store():
     """
     Store management for this ml-git repository.
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/store.py:15]: Adjusted in 27a2458.

File: ml_git/commands/store.py [hunk 5]
```diff
@@ -3,43 +3,24 @@
...
-@click.option('--credentials', help='Profile name for store credentials')
-@click.option('--region', help='Aws region name for S3 bucket')
-@click.option('--type', default='s3h', type=click.Choice(['s3h', 's3', 'azureblobh', 'gdriveh'], case_sensitive=True),
-              help='Store type (s3h, s3, azureblobh, gdriveh ...) [default: s3h]')
-@click.option('--endpoint-url', help='Store endpoint url')
-@click.option('--global', '-g', is_flag=True, default=False, help='Use this option to set configuration at global level')
-@click.option('--verbose', is_flag=True, expose_value=False, callback=set_verbose_mode, help='Debug mode')
-@click.help_option(hidden=True)
-def store_add(**kwargs):
+def store_add(context, **kwargs):
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/store.py]: Adjusted in 27a2458.

File: docs/mlgit_commands.md (hunk 1 above)
Comment (gilMars) [reply docs/mlgit_commands.md]: Adjusted in 27a2458.

File: docs/mlgit_commands.md (hunk 2 above)
Comment (gilMars) [reply docs/mlgit_commands.md]: Adjusted in 27a2458.

File: ml_git/commands/descriptor.py [hunk 6]
```diff
@@ -352,8 +352,11 @@
             '--retry': {'default': 2, 'help': help_msg.RETRY_OPTION},
             '--path': {'default': None, 'help': help_msg.PATH_OPTION},
             '--object': {'default': None, 'help': help_msg.OBJECT_OPTION},
-            '--store-type': {'default': 's3', 'help': help_msg.STORE_TYPE,
-                             'type': click.Choice(['s3', 'gdrive'])},
+            ('--store-type', '--storage-type'): {
+                'default': 's3', 'help': help_msg.STORE_TYPE,
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/descriptor.py]: Adjusted in 27a2458.

**Review by Mun1z — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

File: ml_git/commands/store.py [hunk 7]
```diff
@@ -3,43 +3,14 @@
...
-import click
 from click_didyoumean import DYMGroup
 
-import ml_git.admin as admin
 from ml_git.commands.repository import repository
-from ml_git.commands.utils import set_verbose_mode
 
 
-@repository.group('store', help='Store management for this ml-git repository.', cls=DYMGroup)
+@repository.group('store', help='[DEPRECATED]: Store management for this ml-git repository.', cls=DYMGroup)
```
Comment (andersonpablosilva) [inline ml_git/commands/store.py:11]:
```suggestion
@repository.group('store', help='[DEPRECATED]: Storage management for this ml-git repository.', cls=DYMGroup)
```

File: docs/mlgit_commands.md [hunk 8]
```diff
@@ -216,12 +217,12 @@ Options:
 Examples:
  - To create an entity with s3 as store and importing files from a path of your computer:
```
[the remark this answers is not in the document]
Comment (gilMars) [reply docs/mlgit_commands.md]: Adjusted in b7db910.

File: docs/mlgit_commands.md [hunk 9]
```diff
@@ -216,12 +217,12 @@ Options:
...
 ```
-ml-git dataset create imagenet8 --store-type=s3h --category=computer-vision --category=images --version=0 --import='/path/to/dataset' --mutability=strict
+ml-git dataset create imagenet8 --storage-type=s3h --category=computer-vision --category=images --version=0 --import='/path/to/dataset' --mutability=strict
 ```
 
 - To create an entity with s3 as store and importing files from a google drive URL:
```
[the remark this answers is not in the document]
Comment (gilMars) [reply docs/mlgit_commands.md]: Adjusted in b7db910.

File: docs/mlgit_commands.md [hunk 10]
```diff
@@ -872,12 +875,14 @@ Example:
...
+**Note: Command deprecated, use storage instead store.**
 
 </details>
 
 <details>
-<summary><code> ml-git repository store del </code></summary>
+<summary><code> ml-git repository store del </code>(DEPRECATED)</summary>
 <br>
 
 ```
```
[the remark this answers is not in the document]
Comment (gilMars) [reply docs/mlgit_commands.md:888]: Adjusted in b7db910.

File: ml_git/commands/help_msg.py [hunk 11]
```diff
@@ -37,15 +37,15 @@
 REGION_OPTION = 'AWS region name [default: us-east-1].'
 PATH_OPTION = 'Store folder path.'
 OBJECT_OPTION = 'Filename in store.'
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/help_msg.py]: Adjusted in b7db910.

File: ml_git/commands/help_msg.py [hunk 12]
```diff
@@ -37,15 +37,15 @@
 REGION_OPTION = 'AWS region name [default: us-east-1].'
 PATH_OPTION = 'Store folder path.'
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/help_msg.py]: Adjusted in b7db910.

File: ml_git/commands/help_msg.py [hunk 13]
```diff
@@ -37,15 +37,15 @@
...
                   ' the existence of all contained IPLD links associated.'
 PARANOID_OPTION = 'Adds an additional step that will download all ' \
                   'IPLD and its associated IPLD links to verify the content by ' \
                   'computing the multihash of all these.'
 CATEGORY_OPTION = 'Artifact\'s category name.'
-STORE_TYPE = 'Data store type [default: s3h].'
+STORAGE_TYPE = 'Data storage type [default: s3h].'
 VERSION_NUMBER = 'Number of artifact version.'
 IMPORT_OPTION = 'Path to be imported to the project.'
 WIZARD_CONFIG = 'If specified, ask interactive questions. at console for git & store configurations.'
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/help_msg.py]: Adjusted in b7db910.

File: ml_git/commands/storage.py [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+    Storage management for this ml-git repository.
+    """
+    pass
+
+
+def check_deprecated_command(context):
+    group_name = context.parent.command.name
+    deprecated_group = 'store'
+    if group_name == deprecated_group:
+        print('[WARNING]: Deprecated command, take a look at commands documentation.')
```
[the remark this answers is not in the document]
Comment (gilMars) [reply ml_git/commands/storage.py]: Adjusted in b7db910.

File: ml_git/commands/store.py (hunk 7 above)
Comment (gilMars) [reply ml_git/commands/store.py]: Adjusted in b7db910.

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**